lib/retrieveForm.js

```diff
@@ -54,6 +54,23 @@ export async function retrieveForm(publicServiceId, formId) {
   const sourceBindings = results
         .reduce((acc, b) => [...acc, ...b]);
 
+  // Check whether a user chose "YourEurope" as their publication channel
+  const publicationChannelQuery = `
+    ${PREFIXES}
+
+    ASK {
+      ${sparqlEscapeUri(serviceUri)} a ${type} ;
+        <https://productencatalogus.data.vlaanderen.be/ns/ipdc-lpdc#publicationMedium> <https://productencatalogus.data.vlaanderen.be/id/concept/PublicatieKanaal/YourEurope> .
+    }
+  `;
+  let isYourEurope = (await querySudo(publicationChannelQuery)).boolean;
+
+  // If a user chooses "YourEurope" as their publication channel, load 
+  // the relevants snippets into the content form that render the English fields obligatory.
+  if (FORM_MAPPING[formId] === "content" && isYourEurope) {
+    const english_requirement_form_snippets = fs.readFileSync(`/config/${FORM_MAPPING[formId]}/add-english-requirement.ttl`, 'utf8');
+    form += english_requirement_form_snippets;
```

Comment:
We don't have a lint rule to force it, but we would prefer to use camelCase naming for consistency.